src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+import { configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import "jest-enzyme";
```

Comment:
This setup is inevitably what I often end up doing for clients, but the same authors also publish a dedicated environment to hand-wave this whole setup file away:

https://github.com/FormidableLabs/enzyme-matchers/tree/master/packages/jest-environment-enzyme

I have mixed feelings on whether it's necessary, but just FYI.

Reply:
Oh, interesting! I didn't know that.

I think I prefer explicit in this case, especially because `setupTests.js` is a "magic" file used by react-scripts. Having an example of this file will serve to show people where they can put code to further customize the test environment.

Reply:
That's good to know. I'm not familiar with the magic that comes with CRA so you're likely making the right call.